pkg/Microsoft.Private.PackageBaseline/project.json

```diff
@@ -0,0 +1,246 @@
+{
+  "frameworks": {
+    "netstandard1.6": {
```

Comment:
I suppose we plan to generate this manually every time we ship a netstandard version?